releases/v19.2.0-rc.1.md

```diff
@@ -0,0 +1,75 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-rc.1.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.0-rc.1.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.0-rc.1
+~~~
+
+### SQL language changes
+
+- `INTERVAL(6)` is now supported as an alias for [`INTERVAL`](../v19.2/interval.html), for compatibility with third-party tools. [#41447][#41447]
+- Added the `pg_available_extensions` table, for compatibility with third-party tools. [#41447][#41447]
+- The `pg_attdef()` and `pg_get_indexdef()` [built-in functions](../v19.2/functions-and-operators.html) no longer include type annotations on default expressions, and `pg_get_indexdef()` now includes the index method in the returned `CREATE INDEX` statement, for compatibility with third-party tools. [#41447][#41447]
+
+### Bug fixes
+
+- The `generate_subscripts()` [built-in function](../v19.2/functions-and-operators.html) now behaves properly on `int2vector` and `oidvector` types. [#41447][#41447]
+- Fixed a bug causing the `cluster_logical_timestamp()` [built-in function](../v19.2/functions-and-operators.html) to sometimes return incorrect results. [#41439][#41439]
+- Prevented an unlikely race condition where a WAL pending recycling could be deleted, leading to a crash. This bug was present in past 19.2 alphas/betas only, not in any earlier release. [#41523][#41523]
+
+### Performance improvements
+
+- Metadata queries that use the `'string'::regclass` form are now much more efficient in the presence of a large number of tables. [#41447][#41447]
+- The `pg_get_coldesc()` [built-in function](../v19.2/functions-and-operators.html) now retrieves comments more efficiently. [#41447][#41447]
+
+### Doc updates
+
+- Clarified epoch-based vs. expiration-based leases in the [Replication Layer](../v19.2/architecture/replication-layer.html#epoch-based-leases-table-data) architecture documentation. [#5578][#5578]
+- Updated the list of [Enterprise features](../v19.2/enterprise-licensing.html). [#5599][#5599]
+- Documented the new [`--geo-partitioned-replicas` flag of the `cockroach demo` command](../v19.2/cockroach-demo.html#run-cockroach-demo-with-geo-partitioned-replicas), which starts a 9-node in-memory cluster with the [Geo-Partitioned Replicas](../v19.2/topology-geo-partitioned-replicas.html) data topology applied to the `movr` database. This lets you quickly simulate one of the most effective data topology patterns for getting low latency in a multi-region cluster. [#5587][#5587]  
+
+### Contributors
+
+This release includes 12 merged PRs by 10 authors. We would like to thank the following contributor from the CockroachDB community:
+
+- Aayush Shah (first-time contributor)
```

Comment:
Aayush is an intern. Do we normally include employees on this list?

Reply:
Nope. Removed. Thanks.

Reply:
Nope. Removed. Thanks.